plenum/server/request_handlers/handler_interfaces/read_request_handler.py

```diff
@@ -14,11 +14,9 @@ class ReadRequestHandler(RequestHandler):
     def __init__(self, database_manager: DatabaseManager, txn_type, ledger_id):
         super().__init__(database_manager, txn_type, ledger_id)
 
-    @abstractmethod
     def static_validation(self, request: Request):
```

Comment:
`dynamic_validation` needs to be removed from `ReadRequestHandler` at all, but I think it makes sense to keep `static_validation` abstract to force implementation to think about it.